frontend/src/features/tracker/modules/pipeline/components/PipelineTracker.tsx

```
@@ -0,0 +1,255 @@
+        } catch (error) {
+          console.error("Failed to save pipeline item", error);
+        }
+      }, 400);
+      saveTimeoutRef.current.set(item.id, timeoutId);
+    },
+    [handleSavePipelineItem]
+  );
+
+  useEffect(() => {
+    return () => {
+      saveTimeoutRef.current.forEach((timeoutId) => window.clearTimeout(timeoutId));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)